tests/sync/test_client.py

```diff
@@ -63,12 +64,14 @@ def test_advertise_should_result_in_peer_connections(mock_server):
     ]
 
     client = TChannelSyncClient('test-client')
-    result = client.advertise(routers)
+    future = client.advertise(routers)
+
+    result = future.result()
 
-    assert result.header == ""
-    # @todo https://github.com/uber/tchannel-python/issues/34
-    assert result.body == json.dumps(body)
-    assert client._async_client.peers.hosts == routers
+    assert result.headers == {'as': 'json'}
```

Comment:
Figured out that this is because advertise still uses the old client api/response. I'm going to fix that but it will break advertisement with old TChannel instances :/